GWAS/LMM.ipynb

```diff
@@ -1208,6 +1209,156 @@
     "1. A file with association results for each chromosome (Note: this are given in regard to Allele 2)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "## GMMAT workflow implementation\n",
+    "Documentation can be found [here](https://github.com/hanchenphd/GMMAT/blob/master/inst/doc/GMMAT.pdf)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "### Step1: Creation of the GRM "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "#Calculate standardized GRM using GEMMA\n",
+    "[gemma_grm]\n",
+    "input:bfile\n",
+    "output:f'{cwd}/{bfile:bn}.sXX.txt'\n",
+    "task: trunk_workers = 1, trunk_size = job_size, walltime = '2h', mem = '6G', cores = 1, tags = f'{step_name}_{_output[0]:bn}'\n",
+    "bash: container=container_lmm, expand = \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout' \n",
+    "    gemma \\\n",
+    "    -bfile ${_input:n} \\\n",
+    "    -gk 2 \\\n",
+    "    -o ${_output:bnn}\n",
+    "bash: container=container_lmm,expand='${ }', stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout'\n",
```

Comment:
@yuliu426 there is an option in `gemma` to specify output directory in the command. Please try to use that option and remove this addition step to move files around. 

Even if you want to move in a separate command, because the language is both bash, you can just put an `mv` command (not `cp`!) at the end of the last command without having to use another `bash` action. But let's use output dir option in gemma anyways